repos/fdbt-site/src/pages/api/apiUtils/userData.ts

```diff
@@ -202,51 +195,32 @@ export const getBasePeriodTicketAttributes = (
     const multipleProductAttribute = getSessionAttribute(req, MULTIPLE_PRODUCT_ATTRIBUTE);
     const periodExpiryAttributeInfo = getSessionAttribute(req, PERIOD_EXPIRY_ATTRIBUTE);
 
-    if (!operatorAttribute?.name || isSalesOfferPackageWithErrors(salesOfferPackages) || !salesOfferPackages) {
-        throw new Error(`Could not create ${ticketType} ticket json. BasePeriodTicket attributes could not be found.`);
-    }
-
-    const { name } = operatorAttribute;
-
-    let productDetailsList: ProductDetails[];
-
-    if (!periodExpiryAttributeInfo || !isPeriodExpiry(periodExpiryAttributeInfo)) {
-        throw new Error('Could not create ticket json. Period expiry attribute data problem.');
-    }
-
-    if (!multipleProductAttribute) {
-        const productsAttribute = getSessionAttribute(req, PRODUCT_DETAILS_ATTRIBUTE);
-        if (!productsAttribute || (productsAttribute && !isProductData(productsAttribute))) {
-            throw new Error('productsAttribute was not product data');
-        }
-
-        const { products } = productsAttribute;
-
-        const { productValidity, productEndTime } = periodExpiryAttributeInfo;
-
-        if (isProductWithSalesOfferPackages(salesOfferPackages)) {
-            throw new Error('Could not create geo zone ticket json. Sales offer package info incorrect type.');
-        }
-
-        productDetailsList = products.map(product => ({
-            productName: product.productName,
-            productPrice: product.productPrice,
-            productDuration: product.productDuration,
-            productValidity,
-            productEndTime,
-            salesOfferPackages,
-        }));
-    } else {
-        if (isSalesOfferPackages(salesOfferPackages)) {
-            throw new Error('Could not create geo zone ticket json. Product Sales offer package info incorrect type.');
-        }
-        productDetailsList = getProductsAndSalesOfferPackages(
+    if (
+        !operatorAttribute?.name ||
+        isSalesOfferPackages(salesOfferPackages) ||
+        isSalesOfferPackageWithErrors(salesOfferPackages) ||
+        !salesOfferPackages ||
+        !multipleProductAttribute ||
+        !periodExpiryAttributeInfo ||
+        !isPeriodExpiry(periodExpiryAttributeInfo)
+    ) {
+        logger.error('Attributes missing / incorrect', {
```

Comment:
Do we want to keep this logging in? If so, should we apply the same pattern to other errors?

Reply:
This one is particularly bespoke because we could do with knowing which things are missing / incorrect. There are likely some other areas of the code which could do with this, but I think its tech debt rather than something we should do as part of this ticket.